### PR TITLE
BMI added

### LINKED ISSUE #22: BMI Calculator
### Calculator Functions

The objective of this JavaScript project is to wire up a very basic JavaScript BMI Calculator.
Key concepts covered :-
1.) document.querySelector()
2.) event listeners
3.) parseInt()
4.) toFixed()
5.) innerHTML

### Contribution Event

GSSOC22

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/vasu-1/CalcHub/blob/main/.github/ContributingGuidelines.md) & [Code of conduct](https://github.com/vasu-1/CalcHub/blob/main/.github/CODE_OF_CONDUCT.MD) of this project.

### PR DESCRIPTION
**Pull request by Kushal-2002**
## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
Fixes #22 
I have added a BMI calculator using html, css and javascript.

<hr>
 
## Checklist ✅

- [✅ ] I follow [Contributing Guidelines](https://github.com/vasu-1/CalcHub/blob/main/.github/ContributingGuidelines.md) & [Code of conduct](https://github.com/vasu-1/CalcHub/blob/main/.github/CODE_OF_CONDUCT.MD) of this project.
- [✅ ] I have performed a self-review of my own code or work.
- [✅ ] I have commented my code, particularly in hard-to-understand areas.
- [✅ ] My changes generates no new warnings.
- [✅ ] I'm GSSOC'22 contributor
- [ ✅] Have you added a working video of your calculator ?

<hr>

## Screenshots / Gif (Optional) 📸

BMI Calculator - Google Chrome 3_1_2022 1_19_40 PM

<hr>

### CODE REVIEW

**Comment by vasu-1** [comment]
1. mention the issue number
2. rename the file `readme.md.txt` to `readme.md`
3. add necessary details in the readme
4. add your calculator link in the main readme
5. Also add screenshot to readme

**Comment by Kushal-2002** [comment]
@vasu-1 I have made the necessary changes in the readme file. Can you please tell me  how can I add calculator link in main readme?

**Comment by vasu-1** [comment]
> @vasu-1 I have made the necessary changes in the readme file. Can you please tell me how can I add calculator link in main readme?

That i will do from my end

**Comment by vasu-1** [comment]
delete `readme.md.txt`
And use some markdown in your `readme`
Also don't put(upload) binary(image) file in repo instead you can directly copy the image and paste in readme.md file

**Comment by Kushal-2002** [comment]
Please can you review and merge my pull request for issue #22.

**Comment by vasu-1** [comment]
> delete `readme.md.txt` And use some markdown in your `readme` Also don't put(upload) binary(image) file in repo instead you can directly copy the image and paste in readme.md file
Do this changes

**Comment by Kushal-2002** [comment]
Thank you for the instructions. I have made all the changes and updated my files.

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Binary files need to be deleted !!!
There is still one image file

**Comment by vasu-1** [comment]
Also add your file link in main readme.md
Make sure to update your cloned repo

**Comment by Kushal-2002** [comment]
@vasu-1 I have deleted the binary image file and also added my calculator link in the main Readme file.

**Review by mansi-garg-12 — CHANGES_REQUESTED** [review]
Please remove the .png file and add it on README.md or past a link of google drive.

**Comment by vasu-1** [comment]
image

Image file is still there,
Please remove it
And upload it directly by simply copy and paste in readme

**Comment by Kushal-2002** [comment]
I have removed the .png file and added a google drive image link in the main README.md

File: Calculators/BMI_Calculator/readme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# BMI Calculator
+## I have added the BMI calculator.
```
Comment (vasu-1) [inline Calculators/BMI_Calculator/readme.md:2]: you need to add the image here !
And you can just directly copy your image and paste it in readme file itself.. just like text copy and paste

Comment (Kushal-2002) [reply Calculators/BMI_Calculator/readme.md:2]: I have tried to copy and paste my image several times but it is not happening. So I have added the google drive link.

**Comment by Kushal-2002** [comment]
I have added the image to my readme.md file and deleted it from previous place. But for the image I am unable to do direct copy and paste so I have to added the google drive link.

**Review by mansi-garg-12 — APPROVED** [review]
checked

**Comment by vasu-1** [comment]
Sorry for that !
But there are lot's of conflicts !
I accidently deleted the readme.md

What you can do is, you can just close this pr, delete this cloned repo and again clone it, and than make the folder and do the changes,
After that, make another pr for the same !!!!!!!!!!!!!!

if you have any queries reach out to me at discord !!

**Comment by vasu-1** [comment]
Please make another pr !

**Comment by Kushal-2002** [comment]
@vasu-1 do I need to make a new issue and make PR over there? Also please tell me how can I reach out to you via discord?

**Pull request closed without merging**